### DISCUSSION TITLE
Discord: Breaking Changes notifications

### DISCUSSION THREAD

**Discussion by chris400** [community-scripts/ProxmoxVE#1031] [Ideas]
As a user I would like to be notified of any breaking changes.
Currently I would need to take a look at discussions regularly and manually.

It would be great to be notified specifically for any breaking changes as in https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=label%3A%22breaking+change%22 
Github Webhooks should be able to do that:
https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=labeled#discussion

You could use a separate channel in Discord or separate role for these notifictions.

**Comment by MickLesk** [comment]
@newzealandpaul
